source/CoordinateConversion/CoordinateConversionLibrary/Views/InputCoordinateConversionView.xaml

```diff
@@ -11,6 +11,7 @@
         <ResourceDictionary>
             <ResourceDictionary.MergedDictionaries>
                 <ResourceDictionary Source="/CoordinateConversionLibrary;component/MAResourceDictionary.xaml" />
+                <ResourceDictionary Source="pack://application:,,,/ArcGIS.Desktop.Framework;component\Themes\Default.xaml"/>
```

Comment:
Just checking - is this the line/dependency that didn't work because no Pro assemblies in this project?